build/__variables.sh

```diff
@@ -79,6 +80,7 @@ declare -r ACR_BUILD_VSO_FOCAL_IMAGE_NAME="$ACR_BUILD_IMAGES_REPO:vso-ubuntu-foc
 declare -r ACR_BUILD_VSO_BULLSEYE_IMAGE_NAME="$ACR_BUILD_IMAGES_REPO:vso-debian-bullseye"
 
 declare -r BASE_IMAGES_REPO="$ACR_DEV_NAME/public/oryx/base"
```

Comment:
Nit comment, so please feel free to ignore if you believe it's not worth it/going to create a lot of noise.
Now that we created the STAGING equivalent, should we rename this one to BASE_IMAGES_PRODUCTION_REPO or BASE_IMAGES_PUBLIC_REPO?

Reply:
https://github.com/microsoft/Oryx/pull/1643/commits/718ed1ba5d4cf201b8f204c6007b7d21cf27143c